t/t6042-merge-rename-corner-cases.sh

```diff
@@ -575,4 +575,45 @@ test_expect_success 'rename/rename/add-dest merge still knows about conflicting
 	test ! -f c
 '
 
+test_expect_success 'move file/sparse-checkout/merge should not delete moved file' '
+	git rm -rf . &&
+	git clean -fdqx &&
+	rm -rf .git &&
+	git init &&
```

Comment:
This is unnecessarily expensive. If you want a fresh worktree, just use `git init sparse-test`